src/components/Footer.jsx

```diff
@@ -80,16 +79,18 @@ class Footer extends React.Component {
     sendTrackEvent(eventName, properties);
   }
 
-  CCPADialogCloseHandler() {
-    this.setState({
-      showCCPADialog: false,
-    });
-  }
-
-  CCPADialogOpen() {
-    this.setState({
-      showCCPADialog: true,
-    });
+  toggleConsentModal() {
+    window.OneTrust?.ToggleInfoDisplay();
+    const { host } = new URL(`${MARKETING_BASE_URL}`);
+    // Use global domain (`.edx.org`) without the subdomain
+    const hostParts = host.split('.');
+    const domain = hostParts.length > 2
+      ? `.${hostParts.slice(-2).join('.')}`
+      : host;
+    const cookieOptions = host.startsWith('localhost')
```

Comment:
Since from my understanding the URL.host just returns a string 'edx.org', you never need to check this condition.

Reply:
You are correct. This logic was taken from what was previously the CCPADialog. I'm not sure if they used this cookieOptions logic to test in localhost, however I do agree that this is unnecessary and should be deleted.